safe/report/extractors/composer.py

```diff
@@ -391,3 +399,109 @@ def qgis_composer_extractor(impact_report, component_metadata):
     }
     context.substitution_map = substitution_map
     return context
+
+
+def qgis_composer_infographic_extractor(impact_report, component_metadata):
+    """Extract composer context specific for infographic template.
+
+    This method extract necessary context for a given impact report and
+    component metadata and save the context so it can be used in composer
+    rendering phase
+
+    :param impact_report: the impact report that acts as a proxy to fetch
+        all the data that extractor needed
+    :type impact_report: safe.report.impact_report.ImpactReport
+
+    :param component_metadata: the component metadata. Used to obtain
+        information about the component we want to render
+    :type component_metadata: safe.report.report_metadata.
+        ReportComponentsMetadata
+
+    :return: context for rendering phase
+    :rtype: dict
+
+    .. versionadded:: 4.2
+    """
+    qgis_context = impact_report.qgis_composition_context
+    inasafe_context = impact_report.inasafe_context
```

Comment:
__[flake8]__
* [F841] local variable 'inasafe_context' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->